lib/document/DocumentFactory.js

```diff
@@ -207,6 +210,8 @@ class DocumentFactory {
         $type: document.getType(),
         $dataContractId: document.getDataContractId(),
         $revision: document.getRevision() + 1,
+        $createdAt: document.getCreatedAt().getTime(),
```

Comment:
Replace Transition doesn't need `createdAt` we do not update it. We should add `updatedAt` only in case if it requires in the contract.